applications/plugins/SofaCUDA/CMakeLists.txt

```diff
@@ -267,9 +266,8 @@ if(Sofa.GUI.Qt_FOUND)
 endif()
 
 find_package(SofaDistanceGrid QUIET)
-sofa_find_package(MiniFlowVR QUIET)
-
 if(SofaDistanceGrid_FOUND)
+    sofa_find_package(MiniFlowVR QUIET)
```

Comment:
sure about this one? It is only used if DistanceGrid is on

Reply:
seems miniflowvr is only needed for a stuff in sofadistancegrid so thats why I moved it in the test
Useless to search for it if Sofadistancegrid is not enabled

Reply:
ah yes sorry I thought you did the opposite... need more coffee before morning review 😆